ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -183,67 +143,32 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
     this.setState({ isMarkdownLoaded: true });
   };
 
-  private renderItems = (potServiceMetadata: ServiceMetadataState) => {
-    if (
-      pot.isSome(potServiceMetadata) &&
-      potServiceMetadata.value?.description
-    ) {
-      return (
-        <>
-          <Markdown
-            animated={true}
-            onLoadEnd={this.onMarkdownEnd}
-            onError={this.onMarkdownEnd}
-          >
-            {potServiceMetadata.value.description}
-          </Markdown>
-          <View spacer={true} large={true} />
-        </>
-      );
-    }
-    return null;
-  };
-
-  private renderServiceMetaData(potServiceMetadata: ServiceMetadataState) {
-    if (pot.isSome(potServiceMetadata) && potServiceMetadata.value) {
-      const metadata = potServiceMetadata.value;
-      const service = this.service;
-
-      return (
-        <>
-          <TosAndPrivacyBox
-            tosUrl={metadata.tos_url}
-            privacyUrl={metadata.privacy_url}
-          />
-          <View spacer={true} large={true} />
-
-          <ContactPreferencesToggles
-            serviceId={service.service_id}
-            channels={service.available_notification_channels}
-          />
-          <View spacer={true} large={true} />
-
-          <ServiceMetadata
-            servicesMetadata={metadata}
-            organizationFiscalCode={service.organization_fiscal_code}
-            getItemOnPress={handleItemOnPress}
-            serviceId={service.service_id}
-            isDebugModeEnabled={this.props.isDebugModeEnabled}
-          />
-        </>
-      );
-    }
-    return null;
-  }
-
   public render() {
     const { service, serviceId } = this;
 
     // collect the service metadata
     const potServiceMetadata =
       this.props.content.servicesMetadata.byId[serviceId] || pot.none;
 
+    const metadata =
```

Comment:
```suggestion
    const metadata = pot.toOption(potServiceMetadata).toUndefined();
```
The same but using a pot transformation
What do you think?